### DISCUSSION TITLE
2006/08/16/labview-%E7%A8%8B%E5%BA%8F%E4%B8%AD%E7%9A%84%E7%BA%BF%E7%A8%8B-2-labview-%E7%9A%84%E6%89%A7%E8%A1%8C%E7%B3%BB%E7%BB%9F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#108] [Announcements]
# 2006/08/16/labview-%E7%A8%8B%E5%BA%8F%E4%B8%AD%E7%9A%84%E7%BA%BF%E7%A8%8B-2-labview-%E7%9A%84%E6%89%A7%E8%A1%8C%E7%B3%BB%E7%BB%9F

二、LabVIEW 的执行系统

https://blog.qizhen.xyz/2006/08/16/labview-%E7%A8%8B%E5%BA%8F%E4%B8%AD%E7%9A%84%E7%BA%BF%E7%A8%8B-2-labview-%E7%9A%84%E6%89%A7%E8%A1%8C%E7%B3%BB%E7%BB%9F

**Comment by ruanqizhen** [comment]
之前博客下的留言：


红梅
2009年05月8日 10:06 编辑
写的很好，顶

回复
Qizhen
2007年01月11日 15:08 编辑
标准执行系统

回复
鑫
2007年01月11日 10:45 编辑
我有一个subVI是向串口发控制字,那应该是设为Instrument I/O,还是data acquisition

回复
媛
2006年08月20日 21:46 编辑
呵呵，好炫的空间，过来踩踩

回复
Zhen
2006年08月20日 14:19 编辑
看了您的文章收获很大，发现自己编的程序还很多需要改进的地方，谢谢

回复
毅
2006年08月18日 16:34 编辑
这么复杂
